docs/secrets.md

```diff
@@ -71,3 +71,20 @@ The secrets are made available in the process context running the command script
 :::{note}
 This feature is only available when using the local or grid executors (Slurm, Grid Engine, etc). The AWS Batch executor allows the use of secrets when deploying the pipeline execution via [Nextflow Tower](https://seqera.io/blog/pipeline-secrets-secure-handling-of-sensitive-information-in-tower/).
 :::
+
+## Pipeline script
+
+:::{versionadded} 23.09.0-edge
+:::
+
+Secrets can be accessed in the pipeline script using the `secrets` variable. For example:
+
+```groovy
+workflow.onComplete {
+    println("The secret is: ${secrets.MY_SECRET}")
+}
+```
+
+:::{note}
+This feature is only available when using the local or grid executors (Slurm, Grid Engine, etc). The AWS Batch executor allows the use of secrets when deploying the pipeline execution via [Nextflow Tower](https://seqera.io/blog/pipeline-secrets-secure-handling-of-sensitive-information-in-tower/).
```

Comment:
Should not this work also with Batch when using xpack?

Reply:
I was going to say, it should be "Seqera Platform" instead of Nextflow Tower

Reply:
Right 👉  0d1dad3f